dialogue-core/src/main/java/com/palantir/dialogue/core/Channels.java

```diff
@@ -49,6 +48,7 @@ public static Channel create(
                 .map(channel -> new TracedChannel(channel, "Dialogue-http-request"))
                 .map(ContentDecodingChannel::new)
                 .map(concurrencyLimiter(config))
+                .map(channel -> new FixedLimitedChannel(channel, MAX_REQUESTS_PER_CHANNEL))
```

Comment:
would you mind dropping this into the simulator Strategy enum too please?

Reply:
good call. Perhaps we should overload channels with a method that takes clocks so we can keep them in sync?

Reply:
Yep I think my facade is gonna include a clock